Testscripts/Linux/gpu-tensorflow.sh

```diff
@@ -77,59 +83,74 @@ Prepare_Test_Dependencies()
 
     LogMsg "Install tensorflow-gpu $TensorflowVersion..."
     python -m pip install --upgrade pip
+    LogMsg "Ensured the latest pip version"
     if [ x"$TensorflowVersion" != "xlatest" ]; then
         pip install --upgrade "tensorflow-gpu==$TensorflowVersion"
+        LogMsg "Installed the latest tensorflow-gpu $TensorflowVersion"
     else
         pip install --upgrade tf-nightly-gpu
+        LogMsg "Installed the tensorflow-gpu nightly build"
     fi
 
     LogMsg "Install $CudnnPackage..."
     wget -t 5 "$CudnnPackage" -O cuda.tgz -o download_cudnn.log
     tar -zxf cuda.tgz
     if [ $? -ne 0 ]; then
-        LogErr "Install CuDNN $CudnnPackage failed"
+        LogErr "Failed CuDNN $CudnnPackage installation"
         SetTestStateAborted
         exit 1
+    else
+        LogMsg "Successfully installed CuDNN $CudnnPackage"
     fi
+
     sudo cp -p cuda/lib64/libcudnn* /usr/lib/x86_64-linux-gnu
     if [ $? -ne 0 ]; then
-        LogErr "Copy cuda/lib64/libcudnn* to /usr/lib/x86_64-linux-gnu failed"
+        LogErr "Failed to copy cuda/lib64/libcudnn* to /usr/lib/x86_64-linux-gnu"
         SetTestStateAborted
         exit 1
+    else
+        LogMsg "Successfully copied cuda/lib64/libcudnn* to /usr/lib/x86_64-linux-gnu"
     fi
 
     echo -e "import tensorflow; print(tensorflow.__version__)" | python
     if [ $? -ne 0 ]; then
-        LogErr "Failed to install tensorflow-gpu $TensorflowVersion"
+        LogErr "Failed to import tensorflow-gpu $TensorflowVersion"
         SetTestStateAborted
         exit 1
+    else
+        LogMsg "Successfully import tensorflow-gpu $TensorflowVersion"
     fi
 
     git clone $BenchmarkTool
     if [ $? -ne 0 ]; then
-        LogErr "Get benchmarks tool from $BenchmarkTool failed"
+        LogErr "Failed to clone out the benchmark tool, $BenchmarkTool"
         SetTestStateAborted
         exit 1
+    else
+        LogMsg "Successfully cloned the benchmark tool, $BenchmarkTool"
     fi
+
     LogMsg "Install dependencies and tensorflow-gpu finished"
 
     export PATH=$(ls -d /usr/local/cuda-*)/bin${PATH:+:${PATH}}
     export LD_LIBRARY_PATH=/usr/local/cuda/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 
-    # Try to run the benchmarks test by default value.
+    # Try to run the benchmark test by default value.
     python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py
     if [ $? -ne 0 ]; then
-        LogErr "Try to run benchmarks test failed"
+        LogErr "Try to run benchmark test failed"
```

Comment:
Log how the benchmark is being called

Reply:
fixed in 9626be3c614ab3d45d8977198984ea0a43941873